clients/drcachesim/scheduler/scheduler.h

```diff
@@ -1163,7 +1163,8 @@ template <typename RecordType, typename ReaderType> class scheduler_tmpl_t {
          * #SCHEDULER_USE_SINGLE_INPUT_ORDINALS, returns the input stream ordinal, except
          * for the case of a single combined-stream input with the passed-in thread id
          * set to INVALID_THREAD_ID (the serial analysis mode for analyzer tools) in
-         * which case the last trace record's tid is returned; otherwise returns the
+         * which case the last trace record's tid as an ordinal (in the order observed
```

Comment:
So "tid as an ordinal" is different than "input stream ordinal", because the latter depends on the possibly arbitrary order in which input streams are assigned an ordinal, whereas the former is the order in which the inputs are seen on the (single) output stream.

Could we simplify this? Maybe use "input stream ordinal" in all cases? Or does anything depend on this behavior?

Reply:
Ah I think the confusion is there's yet another error here: s/serial/online/.  This is online analysis mode where the inputs are all mixed together coming across a pipe so the scheduler only sees a single input stream but wants to pretend there's a separate stream per input software thread.  Updated to "online" and added a little more explanation of what online is.